src/libstore/globals.hh

```diff
@@ -147,15 +147,14 @@ public:
         "the log to show if a build fails."};
 
     MaxBuildJobsSetting maxBuildJobs{
-        this, 1, "max-jobs",
+        this, getMaxThreads(), "max-jobs",
```

Comment:
A default of `getMaxThreads()` makes documentation generation non-reproducible. It may be better to represent the "auto" value explicitly, e.g. by changing the type of this option to `std::optional<unsigned int>`, with "auto" encoded as `std::nullopt`.